CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
 - Added error handling of file_id with type string passed to `civis.io.civis_file_to_table`. (#454)
+- Updated references from 'master' to 'main' (#460)
```

Comment:
This change should belong to the "Changed" section just below.

Reply:
Got it!

Reply:
This change is still pending

Reply:
Whoops, thought I had committed that yesterday. Thanks!